packages/flutter_tools/lib/src/ios/migrations/deployment_target_migration.dart

```diff
@@ -31,33 +33,58 @@ class DeploymentTargetMigration extends ProjectMigrator {
       logger.printTrace('AppFrameworkInfo.plist not found, skipping minimum OS version migration.');
     }
 
+    if (_podfile.existsSync()) {
+      processFileLines(_podfile);
+    } else {
+      logger.printTrace('Podfile not found, skipping global platform version migration.');
+    }
+
     return true;
   }
 
   @override
   String migrateFileContents(String fileContents) {
-    const String minimumOSVersionOriginal = '''
+    const String minimumOSVersionOriginal8 = '''
   <key>MinimumOSVersion</key>
   <string>8.0</string>
 ''';
-    const String minimumOSVersionReplacement = '''
+    const String minimumOSVersionOriginal9 = '''
   <key>MinimumOSVersion</key>
   <string>9.0</string>
 ''';
+    const String minimumOSVersionReplacement = '''
+  <key>MinimumOSVersion</key>
+  <string>11.0</string>
+''';
 
-    return fileContents.replaceAll(minimumOSVersionOriginal, minimumOSVersionReplacement);
+    return fileContents
+        .replaceAll(minimumOSVersionOriginal8, minimumOSVersionReplacement)
+        .replaceAll(minimumOSVersionOriginal9, minimumOSVersionReplacement);
   }
 
   @override
   String? migrateLine(String line) {
-    const String deploymentTargetOriginal = 'IPHONEOS_DEPLOYMENT_TARGET = 8.0;';
-    const String deploymentTargetReplacement = 'IPHONEOS_DEPLOYMENT_TARGET = 9.0;';
-    if (line.contains(deploymentTargetOriginal)) {
+    // Xcode project file changes.
+    const String deploymentTargetOriginal8 = 'IPHONEOS_DEPLOYMENT_TARGET = 8.0;';
+    const String deploymentTargetOriginal9 = 'IPHONEOS_DEPLOYMENT_TARGET = 9.0;';
+
+    // Podfile changes.
+    const String podfilePlatformVersionOriginal = "platform :ios, '9.0'";
+
+    if (line.contains(deploymentTargetOriginal8)
+        || line.contains(deploymentTargetOriginal9)
+        || line.contains(podfilePlatformVersionOriginal)) {
       if (!migrationRequired) {
```

Comment:
This is existing code, but I'm a little confused and try to understand this. Why do we still migrate if `!migrationRequired`.

Reply:
Ahh, because `.migrateLine()` is being for each line in `processFileLines`, and if `.migrateLine()` returns non-null (as it does in this codepath), it will set `migrationRequired = true`, so that print will not show for subsequent lines.

Reply:
Exactly, the next line is `// Only print for the first discovered change found.`

Reply:
Ah ok. Sounds familiar, I probably asked the same question before and forgot about it.

When looking at the whole file, it makes sense but it might be a little confusing when doing inline reviewing like this.